### ISSUE TITLE
Default "aria-expanded" attribute should be added

### ISSUE THREAD

**Issue by markvital** [flatlogic/bootstrap-tabcollapse#84]
When accordion is created *aria-expanded* attribute should be added to each heading link:

https://getbootstrap.com/docs/3.3/javascript/#collapse

Now it is only added when I click on accordion panels.

As mentioned in Bootstrap documention:

`Be sure to add aria-expanded to the control element. This attribute explicitly defines the current state of the collapsible element to screen readers and similar assistive technologies. If the collapsible element is closed by default, it should have a value of aria-expanded="false". If you've set the collapsible element to be open by default using the in class, set aria-expanded="true" on the control instead. The plugin will automatically toggle this attribute based on whether or not the collapsible element has been opened or closed.`

```
<div class="panel-heading">
  <h4 class="panel-title">
    <a href="#feature-1-collapse" aria-controls="home" role="tab" data-toggle="collapse" class="js-tabcollapse-panel-heading collapsed" data-parent="#FTOTc-accordion" aria-expanded="false">
          Collapsible group 1
    </a>
  </h4>
</div> 
...
```
